devtools/src/arcs-hcr-list.js

```diff
@@ -0,0 +1,314 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {PolymerElement} from '../deps/@polymer/polymer/polymer-element.js';
+import {html} from '../deps/@polymer/polymer/lib/utils/html-tag.js';
+import {formatTime, MessengerMixin} from './arcs-shared.js';
+import './common/object-explorer.js';
+
+class ArcsHcrList extends MessengerMixin(PolymerElement) {
+  static get template() {
+    return html`
+    <style include="shared-styles">
+      :host {
+        line-height: 24px;
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        display: flex;
+        flex-direction: column;
+      }
+      header {
+        flex-grow: 0;
+      }
+      [main] {
+        flex-grow: 1;
+        overflow-y: auto;
+      }
+      .title {
+        padding: 0 8px;
+        height: 26px;
+        border-bottom: 1px solid var(--mid-gray);
+        background-color: var(--light-gray);
+      }
+      .refresh {
+        -webkit-mask-position: -84px 48px;
+        cursor: pointer;
+        transition: transform .5s;
+        vertical-align: middle;
+      }
+      :host([loading]) .refresh {
+        transform: rotate(1turn);
+      }
+      .content {
+        border-bottom: 1px solid var(--mid-gray);
+        display: flex;
+        flex-direction: column;
+      }
+      object-explorer {
+        margin: 2px 4px;
+      }
+      [name]:not(:empty) {
+        color: var(--devtools-purple);
+        margin-right: 1ch;
+      }
+      [tags]:not(:empty) {
```

Comment:
It seems like you can remove some of that CSS, you don't display tags

Reply:
Removed some unused CSS :)